packages/flutter/lib/src/animation/curves.dart

```diff
@@ -364,7 +364,7 @@ class Cubic extends Curve {
 /// The [ThreePointCubic] class implements third-order Bézier curves, where two
 /// curves share an interior [midpoint] that the curve passes through. If the
 /// control points surrounding the middle point ([b1], and [a2]) are not
-/// colinear with the middle point, then the curve's derivative will have a
+/// collinear with the middle point, then the curve's derivative will have a
```

Comment:
In the Flutter docs, we use US English spelling instead of British English spelling.Can you change these back to `colinear`?